gui/src/pages/WorkspaceInfo.svelte

```diff
@@ -0,0 +1,51 @@
+<script lang="ts">
+  import Layout from '../components/Layout.svelte';
+  import WorkspaceNav from '../components/WorkspaceNav.svelte';
+  import CenterFormPanel from '../components/form/CenterFormPanel.svelte';
+  import { api } from '../lib/api';
+  import Spinner from '../components/Spinner.svelte';
+
+  export let params = { workspace: '' };
+
+  const workspaceId = params.workspace;
+  const workspace = api.workspace(workspaceId);
+  const workspaceRoute = `/workspaces/${encodeURIComponent(workspaceId)}`;
+</script>
+
+<Layout>
+  <WorkspaceNav {workspaceId} active="Dashboard" slot="navbar" />
+  <CenterFormPanel title="Workspace details" backRoute={workspaceRoute}>
+    {#await workspace.describeSelf()}
+      <Spinner />
+    {:then description}
+      <table>
```

Comment:
Please wrap generic tables with `<CheckeredTableWrapper />` for nice styling out of the box.